templates/next/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "template-next",
```

Comment:
is there a way for us to maybe deduplicate the framesjs-starter code?

Reply:
I would like that but that would require a lot of work, so perhaps it is better to split it to another PR.

Reply:
agreed